manifest-list.md

```diff
@@ -118,4 +118,4 @@ Instead they MUST ignore unknown properties.
 }
 ```
 
-[runtime-platform]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc2/config.md#platform
+[runtime-platform2]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc2/config.md#platform
```

Comment:
The current approach to single-file PDF/HTML generation is to [feed all the Markdown into Pandoc][1].  Pandoc  seems to treat them as a single, concatenated file internally and [doesn't like seeing the same reference defined more than once][2], so he's giving this a globally-unique name to avoid conflicting with [the entry in `config.md`][3].

[1]: https://github.com/opencontainers/image-spec/blob/09c5f357046cbe6af98325b3efe3e866912f6938/Makefile#L70-L76
[2]: https://github.com/opencontainers/runtime-spec/pull/626#discussion_r87822930
[3]: https://github.com/opencontainers/image-spec/blame/09c5f357046cbe6af98325b3efe3e866912f6938/config.md#L249